app/routes/resources/case-studies/route.tsx

```
@@ -1,9 +1,12 @@
-import { MetaFunction, useLoaderData } from "@remix-run/react";
 import { json } from "@remix-run/node";
-import { client } from "@tina/__generated__/client";
-import type { CaseStudiesQuery } from "@tina/__generated__/types";
+import { MetaFunction, useLoaderData } from "@remix-run/react";
 import { useTina } from "tinacms/dist/react";
+
 import { CaseStudiesPage } from "./page";
```

ESLint

[error] There should be at least one empty line between import groups

(import/order)